src/components/molecules/VenuePartygoers/VenuePartygoers.tsx

```diff
@@ -1,31 +1,38 @@
 import React, { useMemo } from "react";
 
-import { useSelector } from "hooks/useSelector";
-import { usePartygoers } from "hooks/users";
-
-import { currentVenueSelector, parentVenueSelector } from "utils/selectors";
+import { useRecentWorldUsers } from "hooks/users";
+import { useConnectRelatedVenues } from "hooks/useConnectRelatedVenues";
+import { useVenueId } from "hooks/useVenueId";
 
 import "./VenuePartygoers.scss";
 
 export const VenuePartygoers = () => {
-  const venue = useSelector(currentVenueSelector);
-  const parentVenue = useSelector(parentVenueSelector);
-  const partygoers = usePartygoers();
-
-  const currentVenueTitle = venue?.attendeesTitle ?? "partygoers";
-  const attendeesTitle = parentVenue?.attendeesTitle ?? currentVenueTitle;
-  const venueName = venue?.name;
-
-  const currentVenuePartygoers = useMemo(() => {
-    if (!venueName) return [];
-
-    return partygoers.filter((partygoer) => partygoer.lastSeenIn?.[venueName]);
-  }, [partygoers, venueName]);
-  const numberOfPartygoers = currentVenuePartygoers.length;
-
-  return (
-    <div className="venue-partygoers-container">
-      {numberOfPartygoers} {attendeesTitle} online
-    </div>
-  );
+  const venueId = useVenueId();
+  const {
+    parentVenue,
+    currentVenue,
+    isCurrentVenueLoaded,
+  } = useConnectRelatedVenues({ venueId });
```

Comment:
Had to use it, since we don't have useParentVenue. Will need to refactor this in the future

Reply:
Also, this will be changed, when we have `world` concept implemented. I think `attendeesTitle` should be in the `world entity`

Reply:
agreed, and captured in https://github.com/sparkletown/internal-sparkle-issues/issues/127#issuecomment-764335927